tools/cli/namespaceCommands.go

```diff
@@ -293,6 +293,13 @@ func (d *namespaceCLIImpl) DescribeNamespace(c *cli.Context) {
 	if namespaceID == "" && namespace == "" {
 		ErrorAndExit("At least namespaceId or namespace must be provided.", nil)
 	}
+	if c.GlobalIsSet(FlagNamespace) && namespaceID != "" {
+		ErrorAndExit("Only one of namespaceId or namespace must be provided.", nil)
+	}
+	if namespaceID != "" {
+		namespace = ""
+	}
```

Comment:
`namespace` should always be empty if `namespaceID` is not. I think this `if` can be removed.

Reply:
This is not the case because of the namespace option defaulting to 'default'.